ajax/libs/layui/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "layui",
+  "filename": "layui.js",
+  "version": "2.2.45",
+  "description": "经典模块化前端框架",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/sentsin/layui.git"
+  },
+  "author": "贤心",
```

Comment:
Here should be in English too, you can use info in bower.json from upstream https://github.com/sentsin/layui/blob/master/bower.json#L6.
When you write package.json for cdnjs library. package.json and bower.json from upstream are both good reference :)

Reply:
Done